examples/kmm-sample/iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -307,7 +307,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd \"$SRCROOT/..\"\n./gradlew :shared:packForXCode -PXCODE_CONFIGURATION=${CONFIGURATION}\n";
+			shellScript = "cd \"$SRCROOT/..\"\nANDROID_SDK_ROOT=$HOME/Library/Android/sdk ./gradlew :shared:packForXCode -PXCODE_CONFIGURATION=${CONFIGURATION}\n";
```

Comment:
It worked out locally for me before applying this, so there must be some other mechanism for "injecting" this...but I guess this should ruing anything.

Reply:
still asking for it locally for me, maybe because I'm using the ` includeBuild("../../packages")` which compiles and build the artefacts from XCode rather than pulling them via maven repo ...